mono/metadata/metadata.h

```diff
@@ -10,6 +10,7 @@
 #include <mono/metadata/blob.h>
 #include <mono/metadata/row-indexes.h>
 #include <mono/metadata/image.h>
+#include <mono/metadata/object-forward.h>
```

Comment:
Same as `exception.h` - `metadata.h` is going to be installed, but `object-forward.h` won't be, so anyone using the embedding API will get a broken header.

Reply:
Right, should be fixed by 3b892d45b000f7e9d4ec988dd00beebede1afe9d.